service/history/historyEngine.go

```diff
@@ -1580,11 +1580,16 @@ func (e *historyEngineImpl) RecordActivityTaskStarted(
 			response.ScheduledEvent = scheduledEvent
 			response.ScheduledTimestampOfThisAttempt = common.Int64Ptr(ai.ScheduledTime.UnixNano())
 
+			response.StartedTimestamp = common.Int64Ptr(ai.StartedTime.UnixNano())
```

Comment:
StartedTimestamp can be updated inside AddActivityTaskStartedEvent. Other fields seems to be fine.